### PR TITLE
Article: JavaScript while Loop

### PR DESCRIPTION
**Pull request by nag786**

### CODE REVIEW

**Comment by Rafase282** [comment]
I made a PR against yours, https://github.com/nag786/wiki/pull/1

File: While-loop.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# while Loop
+-------------
+
+The while loop starts by evaluating the condition. If the condition is true, the statement(s) is executed. If the condition is false, the statement(s) is not executed. After that, while loop ends.
```
Comment (alayek) [inline While-loop.md]:
```diff
- the statement(s) is executed
+ the statement(s) is / are executed
```

File: While-loop.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Here is the **syntax** for while loop:
+________
+
+### Syntax:
+
```
Comment (alayek) [inline While-loop.md]: Put code snippets in markdown multi-line code block.

File: While-loop.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+*statement(s):* A statement that is executed as long as the condition evaluates to true.
+
+*condition:* Here, condition is a Boolean expression which is evaluated before each pass through the loop. If this condition evaluates to true, statement(s) is executed. When condition evaluates to false, execution continues with the statement after the while loop.
+
+______
+
+
+### **Example:**
+
```
Comment (alayek) [inline While-loop.md]: Same as above. Use multi-line code block.

File: While-loop.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    5
+    6
+    7
+    8
+    9
+	
+   ___
+   
+   source:[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while)
+
```
Comment (alayek) [inline While-loop.md]: Remove these blank lines.

File: While-loop.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    4
+    5
+    6
+    7
+    8
+    9
+	
+   ___
+   
+   source:[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while)
```
Comment (alayek) [inline While-loop.md]:
```diff
- source:[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while)
+ source:[While Loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while)

**Comment by alayek** [comment]
@nag786 thanks for the article! Please implement the changes suggested (or use @Rafase282 's PR on your repo) and we are good to go!

If you need help, let us know [here](https://gitter.im/FreeCodeCamp/Wiki).

**Comment by abhisekp** [comment]
@nag786 
**File rename:-**
```diff
- While-loop.md
+ JS-while-Loop.md
```
@Rafase282 Let's keep the keywords in lower case and namespace the file with `JS-` prefix

**Comment by Rafase282** [comment]
@abhisekp @nag786 I updated my PR.

**Pull request closed without merging**